src/main/java/seedu/address/model/entry/EntryType.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.model.entry;
```

Comment:
Thanks for splitting the entry types into this class! Makes it easy to id which type it is in our model manager